### DISCUSSION TITLE
Self Hosted

### DISCUSSION THREAD

**Discussion by ali-master** [inngest/inngest#1122] [Q&A]
Hi there

Is there any plan to make the Cloud project self-hostable?
Our service has a lot of sensitive data and we can't publish our long-running jobs data to any cloud-based solution Inngest is what we need but we have no Cloud features.

Thanks

**Comment by darwin67** [comment]
Not at the moment! There are still a couple of larger architecture changes we're planning to go through so self hosting isn't something we'll be supporting in the near term.
We simply don't have the capacity to support issues when they arise.

Feel free to follow this thread. We'll likely post updates about self hosting on a couple different comm channels when we're there.
https://github.com/orgs/inngest/discussions/138
